BEAR SDK v3.2.1 (#1)
andysklyarov/bear_sdk_demo_android: CHANGELOG.md

---

## [3.2.1](https://github.com/bear2b/bear_sdk_demo_android/releases/tag/3.2.0)

#### Added:
* disable using IDFA by default and add ability to re-enable it in config;

> * Show an error for old version of app if asset type unity is not supported;

Reviewer: ```suggestion
* Show an error if the asset type is not supported;
```

* Android Q compatibility (29 API)

> #### Fixes:
> * Requests in app doesn't work after token expiration
> * Screenshot button doesn't work 

Reviewer: ```suggestion
#### Fixes:
* Requests in app doesn't work after token expiration;
* Screenshot button doesn't work;
```
